upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -1190,3 +1190,311 @@ There you can find links to upgrade notes for other versions too.
+        ```diff
+        - type: "[Image]"
+        + type: "[Image!]!"
+        ```
+    - in GQL definition of `AdvertImageDecorator` is fixed attribute for images array.
+        ```diff
+        - image:
+        -     type: "[Image]"
+        + images:
+        +     type: "[Image!]!"
+        ```
+    - All functions in Mutation classes used as alias are renamed by convention with suffix `Mutation`
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Customer\User\CustomerUserMutation` class are renamed functions:
+        ```diff
+        - public function changePassword(Argument $argument, InputValidator $validator): CustomerUser
+        + public function changePasswordMutation(Argument $argument, InputValidator $validator): CustomerUser
+        ```
+        ```diff
+        - public function changePersonalData(Argument $argument, InputValidator $validator): CustomerUser
+        + public function changePersonalDataMutation(Argument $argument, InputValidator $validator): CustomerUser
+        ```
+        ```diff
+        - public function register(Argument $argument, InputValidator $validator): array
+        + public function registerMutation(Argument $argument, InputValidator $validator): array
+        ```
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Login\LoginMutation` class is renamed function:
+        ```diff
+        - public function login(Argument $argument): array
+        + public function loginMutation(Argument $argument): array
+        ```
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Login\RefreshTokensMutation` class is renamed function:
+        ```diff
+        - public function refreshTokens(Argument $argument): array
+        + public function refreshTokensMutation(Argument $argument): array
+        ```
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Logout\LogoutMutation` class is renamed function:
+        ```diff
+        - public function logout(): array
+        + public function logoutMutation(): array
+        ```
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Newsletter\NewsletterMutation` class is renamed function:
+        ```diff
+        - public function newsletterSubscribe(Argument $argument, InputValidator $validator): array
+        + public function newsletterSubscribeMutation(Argument $argument, InputValidator $validator): array
+        ```
+        - in `Shopsys\FrontendApiBundle\Model\Mutation\Order\CreateOrderMutation` class is renamed function:
+        ```diff
+        - public function createOrder(Argument $argument, InputValidator $validator): Order
+        + public function createOrderMutation(Argument $argument, InputValidator $validator): Order
+        ```
+  - in GQL mutation definitions are replaced all old aliases for renamed Mutation functions, e.g.:
+        ```diff
+        - resolve: "@=mutation('create_order', args, validator)"
+        + resolve: "@=mutation('createOrderMutation', args, validator)"
+        ```
```

Comment:
As I see this, there is no simple rule to be applied. You have changed names in multiple ways thus I think you should create list of changes

Reply:
ok, i added all changes